tests/subgraph/mapping-v3-core/mapping-v3-core.test.ts

```diff
@@ -1389,6 +1399,12 @@ describe(`${coreType}: handleProjectUpdated`, () => {
         "artistAddress",
         artistAddress.toHexString()
       );
```

Comment:
Awesome! I think we need to add one more test for the default base URI when a project is added (not updated) right?